seq2seq/evaluator/predictor.py

```diff
@@ -42,3 +43,31 @@ def predict(self, src_seq):
         tgt_id_seq = [other['sequence'][di][0].data[0] for di in range(length)]
         tgt_seq = [self.tgt_vocab.itos[tok] for tok in tgt_id_seq]
         return tgt_seq
+
+    def predict_n(self, src_seq, n=None):
+        """ Make 'n' predictions given `src_seq` as input.
+
+        Args:
+            src_seq (list): list of tokens in source language
+            n (int): number of predicted seqs to return. If None,
+                     it will return just one seq.
+
+        Returns:
+            tgt_seq (list): list of tokens in target language as predicted
+                            by the pre-trained model
+        """
+        src_id_seq = Variable(torch.LongTensor([self.src_vocab.stoi[tok] for tok in src_seq]),
+                              volatile=True).view(1, -1)
+        if torch.cuda.is_available():
+            src_id_seq = src_id_seq.cuda()
+
+        softmax_list, _, other = self.model(src_id_seq, [len(src_seq)])
```

Comment:
Line 59-64 are exactly the same as line 35-40, it'd be better to extract a function for that.

Reply:
I placed this code in a new function called `get_model_features`